client/components/happychat/style.scss

```diff
@@ -205,6 +220,17 @@
 	}
 }
 
+@keyframes happychat-fade-out-right {	// for cases where the left sidebar does not exist (full page views)
+	0% {
+		opacity: 1;
+		transform: translateX(0px);
```

Comment:
You might not even need the 0% rules since they should be there already. Try it without to see what happens.

Reply:
They're not needed but assure best browser support.

Reply:
Ah. In the future, we don't need them. The browsers Calypso functions in all work without it. Not a big deal, though. Nice work!

Reply:
🙌